src/process.c

```diff
@@ -0,0 +1,55 @@
+/**
+ * Source: https://gist.github.com/fclairamb/a16a4237c46440bdb172
```

Comment:
Remark: requested license clarification since not stated - see https://gist.github.com/fclairamb/a16a4237c46440bdb172#gistcomment-3630465

Reply:
https://gist.github.com/fclairamb/a16a4237c46440bdb172#gistcomment-3630499
![Bildschirmfoto-20210213220551-1022x432](https://user-images.githubusercontent.com/3312979/107861693-aed72200-6e47-11eb-8b7d-22959958e6c4.png)